### DISCUSSION TITLE
Customize badge color in Relation Manager

### DISCUSSION THREAD

**Discussion by brain82** [filamentphp/filament#9452] [Ideas]
I like the fact that we're able to add badges to the tabs of relationships via the Relation Manager class. However, it will be great if we can change the color of the badge.

<img width="1873" alt="image" src="https://github.com/filamentphp/filament/assets/6013683/7420c519-a989-4667-87cf-4577ffb3688e">

**Comment by devhoussam1998** [comment]
https://filamentphp.com/docs/3.x/support/enums

**Comment by somzoli** [comment]
Hi, How did you add this badge to the relation manager tabs?

**Comment by devhoussam1998** [comment]
> Hi, How did you add this badge to the relation manager tabs?

https://filamentphp.com/docs/3.x/panels/resources/listing-records#adding-badges-to-filter-tabs

**Comment by gldtn** [comment]
Can you share your tabs code for relationship manager tabs? Do you place it in the view?

**Comment by brain82** [comment]
@gldtn 

<img width="678" alt="image" src="https://github.com/filamentphp/filament/assets/6013683/f74de58d-6721-465f-927c-c5e20d07b03b">

**Reply by mattsidnell** [reply]
Thanks so much.  Saved me about a day!!
